src/audio.c

```diff
@@ -1105,9 +1127,9 @@ int audio_alloc(struct audio **ap, struct list *streaml,
 
 	err = stream_alloc(&a->strm, streaml,
 			   stream_prm, &cfg->avt, sdp_sess,
-			   MEDIA_AUDIO, label,
+			   MEDIA_AUDIO, label, true, ptime,
```

Comment:
I dont think that `ptime` should be sent into stream.
ptime is audio specific, and stream is generic.

Reply:
The ptime is passed to jbuf. It is the time difference between two RTP packets. For audio it is the same like the frame size. For Video it is the time how long a video frame is viewed.
In this sense, the packet time is stream specific. No matter if video or audio. Right?